### DISCUSSION TITLE
Google flagged my cockpit as "Deceptive site"

### DISCUSSION THREAD

**Discussion by amartini** [cockpit-project/cockpit#18221] [General]
I have added cockpit to my VPS, and added a url that access it directly. Since two days ago when I open the url I get this message:
![x](https://user-images.githubusercontent.com/894369/213942560-42f829c2-4807-409c-91cf-24872633c102.png)

Anyone had this before? What can I do to fix? Already requested reviewing but don't know how long it takes. (If anyone want the url pm me)

**Comment by garrett** [comment]
I don't think this is actionable by us. We don't even know what browser or OS or version of Cockpit or if you're using a proxy or anything else that you're doing on your site, so there's no way to even begin to really know what's going on.

Google does have more information at the bottom of @
https://support.google.com/chrome/answer/99020?hl=en&co=GENIE.Platform%3DDesktop

And has a reporting tool @ 
https://safebrowsing.google.com/safebrowsing/report_error/?hl=en
